backend/entrypoint.sh

```diff
@@ -14,6 +14,16 @@ echo "Running backend server..."
 
 python manage.py rqworker default 2>&1 | tee /logs/rqworker.log &
 
+echo "Installing crontab..."
+
+if [ -z "$IMAGE_SCAN_SCHEDULE" ]
+then
+    IMAGE_SCAN_SCHEDULE="0 */6 * * *"
+fi
+
+echo "$IMAGE_SCAN_SCHEDULE python3 manage.py scan 2>&1" > crontab
```

Comment:
```suggestion
echo "$IMAGE_SCAN_SCHEDULE python3 manage.py scan" > crontab
```

I suggest to use a simple syntax.